src/org-unit-tree/OrgUnitTree.component.js

```diff
@@ -50,6 +50,7 @@ class OrgUnitTree extends React.Component {
         super(props);
 
         this.state = {
+            useShortNames: props.useShortNames,
```

Comment:
`useShortNames` is not updated within the component, right? then it can be simply used from the props.

Reply:
Totally. Done in https://github.com/EyeSeeTea/d2-ui-components/commit/62085fc06ed